torchgeo/trainers/byol.py

```diff
@@ -401,33 +409,10 @@ def training_step(self, *args: Any, **kwargs: Any) -> Tensor:
         return loss
 
     def validation_step(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
I don't think it makes sense to include validation/test/predict in an SSL trainer. For datasets without labels, there is no easy way to evaluate performance. For datasets with labels, there is no way to know which task we are trying to evaluate (classification, regression, semantic segmentation, etc.)

Reply:
I think this is fine. There could be an infinite number of downstream tasks

Reply:
https://pytorch-lightning.readthedocs.io/en/stable/notebooks/course_UvA-DL/13-contrastive-learning.html defines a validation_step but I can't figure out why